spec/takeaway_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:menu) { double :menu }
+  before(:each) { allow(menu).to receive(:content).and_return({ chicken: 4, lamb: 7, vegetables: 3, latte: 2 }) }
+  let(:takeaway) { Takeaway.new(menu) }
```

Comment:
Use the [explicit `subject`](https://www.relishapp.com/rspec/rspec-core/docs/subject/explicit-subject) here instead of `let`, as you're modifying the implicit `subject` rather than declaring something wholly new.